src/Tickets/Flexible_Tickets/WP_Cli.php

```diff
@@ -67,7 +67,7 @@ public function is_active(): bool {
 	 *
 	 * @return bool Whether the foreign key checks were disabled or not.
 	 */
-	public function maybe_disable_foreign_key_checks( array $args ): bool {
+	public function maybe_disable_foreign_key_checks( array $args = [] ): bool {
```

Comment:
I think we also need a check on if the array key(s) exist(s), since if `$args` is [] then `$args[0]` (and 1) will throw an undefined key warning